components/CurrentPage.php

```diff
@@ -24,9 +24,11 @@ public function __construct(
      */
     public function value(): int
     {
-        return explode(
-            "=",
-            end(explode("/", $this->uri))
-        ) ?? 1;
+        preg_match(
+            "/(page=)([1-9][0-9]*)/",
+            explode("?", $this->uri)[1],
+            $matches
+        );
+        return $matches[2];
```

Comment:
It previously returned an array for some unknown reason. Now the return type is correct.